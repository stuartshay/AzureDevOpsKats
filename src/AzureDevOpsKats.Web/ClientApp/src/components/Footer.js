import React from 'react';
import  {MDBFooter}  from "mdbreact";

const Footer = (props) => {
    return (
      < MDBFooter color={!props.theme?"green":props.theme} style={{ position: "absolute", width: '100%' }}>
        <p className="footer-copyright mb-0 py-3 text-center">
          &copy; {new Date().getFullYear()} Copyright: {props.env} DevOpsKats
        </p>
      </MDBFooter >
    )
}

export default Footer;
