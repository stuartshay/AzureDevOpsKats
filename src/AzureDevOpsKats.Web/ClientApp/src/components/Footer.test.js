import React from 'react';
import Footer from './Footer';
import Enzyme, { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';

Enzyme.configure({ adapter: new Adapter() });

// SnapShot Testing for the Footer component

it('renders correctly enzyme', () => {
    const wrapper = shallow(<Footer />)
  
    expect(toJson(wrapper)).toMatchSnapshot();
});