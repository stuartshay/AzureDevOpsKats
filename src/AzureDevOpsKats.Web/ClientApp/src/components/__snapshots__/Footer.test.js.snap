// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly enzyme 1`] = `
<at
  color="footer-bg"
  style={
    Object {
      "position": "absolute",
      "width": "100%",
    }
  }
  tag="footer"
>
  <p
    className="footer-copyright mb-0 py-3 text-center"
  >
    Â© 
    2022
     Copyright: Azure DevOpsKats
  </p>
</at>
`;
